daemon/src/wallet.rs

```diff
@@ -109,6 +117,26 @@ impl Actor<ElectrumBlockchain, sled::Tree> {
         let client = bdk::electrum_client::Client::new(electrum_rpc_url)
             .context("Failed to initialize Electrum RPC client")?;
 
+        let network_hash = constants::genesis_block(ext_priv_key.network).block_hash();
+        // This needs to be converted to a BlockHash first, as Sha256d hashes are displayed backward
```

Comment:
This is *weird*! Nothing we can do about it though - maybe someone else knows the history better, but judging by the comments in `BlockHash::from_byte_iter`, this is just a bitcoin oddity.